tutorials/cp-aibus-ber-service-instance/cp-aibus-ber-service-instance.md

```diff
@@ -8,7 +8,7 @@ primary_tag: topic>machine-learning
 ---
 
 ## Prerequisites
-- You have created a trial account on SAP BTP: [Get a Free Trial Account on SAP BTP](hcp-create-trial-account)
+- You have created a trial account on SAP BTP: [Get a Free Account on SAP BTP Trial](hcp-create-trial-account)
```

Comment:
According to SAPterm it should be "SAP BTP trial". "SAP BTP Trial" would be correct if used in a title, though.